src/hmem_cuda.c

```diff
@@ -744,6 +751,11 @@ int cuda_host_unregister(void *ptr)
 	return -FI_EIO;
 }
 
+enum cuda_xfer_setting cuda_get_xfer_setting()
+{
+	return hmem_cuda_xfer_setting;
+}
```

Comment:
Why add a wrapper function, versus just defining hmem_cuda_xfer as an extern and reading that directly?

Reply:
I was following the convention that other similar variables used, like `cuda_ipc_enabled` was returned via function `cuda_is_ipc_enabled()`. I do not have preference. Do you perfer extern variable over function?